packages/web/src/scss/components/Modal/_ModalDialog.scss

```diff
@@ -4,10 +4,52 @@
 // 3. We need to set the box-sizing again because the parent element unsets styles using `all: unset`.
 
 @use 'sass:map';
+@use '../../settings/feature-flags';
 @use '../../tools/breakpoint';
 @use '../../tools/typography';
 @use 'theme';
 
+@mixin -uniform-modal-dialog() {
```

Comment:
I thought we kept all mixins and functions in component tools. 🤔

Reply:
Maybe they are here because they are local? 

The problem with local mixins is that we are not able to test them :/

Reply:
Yes, these are just temporary local mixins, not intended for reusing. I don't think that testing them would be any helpful… We will remove them in the next major version anyway.

Reply:
Please, be careful. Not testing something because it will be removed in the future can be problematic. We plan the next major version at the start of the Q2. So this code will be here for a while. Just saying.

Reply:
Well, until a few weeks ago, we didn't have tests in Sass at all… What I'm trying to say here is this code only puts aside a few lines of other code. Plus we can expect it will have a very short life, but that's the less important thing.

A test could only check if the mixin prints exactly what is inside it. It doesn't programmatically generate other code, nor there are any injected dependencies. Thus it behaves deterministic: as long as the theme variables remain the same, it's fine. If they don't, we have a much bigger problem anyway.

To sum up, I get what you mean _in general_. I'm not advising against writing tests. But in this case, I don't see any benefit in testing that a function returning a string actually returns the string—that sounds more like a constant, and I don't think we test constants, do we 🙂.